govuk/sign-in/token/index.js

```diff
@@ -1,30 +1,96 @@
 'use strict';
 
-// const request = require('../../../questionnaire/request-service')();
+const {v4: uuidv4} = require('uuid');
+
+const createJwtService = require('../jwt');
+const {post} = require('../../../questionnaire/request-service')();
+const createIssuerService = require('../issuer');
+const {getSignedInURI} = require('../../../account/utils/getActionURIs');
 
 function createTokenService() {
-    async function getIdToken(options) {
-        /* const opts = {
+    async function getTokenData(authorisationCode) {
+        if (!authorisationCode) {
+            throw Error('Authorisation code is required');
+        }
+
+        const jwtService = createJwtService();
+        const opts = {
+            url: `${process.env.CW_GOVUK_ISSUER_URL}/token`,
```

Comment:
You need to grab this from the mighty issuer!! Can't assume this will always be the endpoint